### ISSUE TITLE
Usable for those outside of Duke?

### ISSUE THREAD

**Issue by csheaff** [mlp6/ultratrack#15]
Hi Mark,

I'm a postdoc at UC-Davis looking to give your code a whirl. Could you tell me if it is intended not only for those at Duke? If so, any tips on creating a probe definitions file that is compatible?

Clay

**Comment by mlp6** [comment]
Yep, wide open for use!  I have a lot of probe definitions for Siemens probes, but those are considered proprietary... any chance you are working under a Siemens MRA thought your parent lab?

**Comment by csheaff** [comment]
Hmm...not in our lab, but an imaging center in our building has Siemens PET/SPECT/CT scanners. Think that might be workable?

**Comment by mlp6** [comment]
Not likely... but I can commit some templates.  What probe type(s) are you looking to use?  Linear?  Curvilinear?  Phased?  Matrix?

**Comment by csheaff** [comment]
Ah, some templates would be great. Linear type would work. Thanks!

**Comment by mlp6** [comment]
I just realized that there is a template probe file in the repository already, but it is mislabeled: `probe_linear.m`.  That should actually be `probe_linear.txt`; it is ultimately read in using `code/uf_txt_to_probe.m` in `do_dyna_scans.m` (line 68).  There might be a path issue where the code tries to find that txt file, but that is the general format it is expecting.

**Comment by csheaff** [comment]
Okay, thanks Mark! Might have another question or two in the future, as I haven't had a chance to look at it recently. Hope you don't mind.

Cheers,
Clay
